specification/paloaltonetworks/resource-manager/PaloAltoNetworks.Cloudngfw/stable/2025-05-23/PaloAltoNetworks.Cloudngfw.json

```diff
@@ -0,0 +1,7608 @@
+      "post": {
+        "tags": [
+          "GlobalRulestack"
+        ],
+        "operationId": "GlobalRulestack_revert",
+        "description": "Revert rulestack configuration",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/GlobalRulestackResource.globalRulestackName"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "There is no content to send for this request, but the headers may be useful. "
+          },
+          "default": {
+            "description": "Common error response for all Azure Resource Manager APIs to return error details for failed operations.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GlobalRulestack_revert_MaximumSet_Gen": {
+            "$ref": "./examples/GlobalRulestack_revert_MaximumSet_Gen.json"
+          },
+          "GlobalRulestack_revert_MinimumSet_Gen": {
+            "$ref": "./examples/GlobalRulestack_revert_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/providers/PaloAltoNetworks.Cloudngfw/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_List",
+        "description": "List the operations for the provider",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Azure operation completed successfully.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Common error response for all Azure Resource Manager APIs to return error details for failed operations.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Operations_List_MaximumSet_Gen": {
+            "$ref": "./examples/Operations_List_MaximumSet_Gen.json"
+          },
+          "Operations_List_MinimumSet_Gen": {
+            "$ref": "./examples/Operations_List_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/PaloAltoNetworks.Cloudngfw/createProductSerialNumber": {
+      "post": {
+        "operationId": "PaloAltoNetworksCloudngfw_createProductSerialNumber",
```

Comment:
This prefix for operation group is conflicting with client name, this caused the JS generation failure, is it possible to rename this operation group like `PaloAltoNetworksCloudngfwOperations`?

Reply:
Thanks for looking into it @MaryGao!

@rohit4637 Can you check if it is possible to rename this operation group?